compiler/ast/parser.nim

```diff
@@ -1158,8 +1181,8 @@ proc primary(p: var Parser, mode: PrimaryMode): ParsedNode =
     # Note 'sigil like' operators are currently not reflected in the grammar
     # and should be removed for Nim 2.0, I don't think anybody uses them.
     let isSigil = isSigilLike(p.tok)
-    result = p.newNode(nkPrefix)
-    var a = p.newNode(nkIdent, p.tok)
+    result = p.newNode(pnkPrefix, p.tok) # TODO: before merge confirm `p.tok` is correct
```

Comment:
yep

Reply:
thanks, noted it locally, will push changes shortly.